### PR TITLE
Feature/ethereum api new methods

### PR DESCRIPTION
**Pull request by ginstagram**
## Description
Ethereum debug & trace API methods

### CODE REVIEW

**Comment by akegaviar** [comment]
@ginstagram conflicts and version please

File: docs/release-notes.md [hunk 1]
```diff
@@ -15,7 +15,9 @@ meta:
 ### What's new
 
 * **Protocols**. [Gnosis Chain clients](/operations/gnosis/clients) updated for The Merge.
-* **Documentation**. Gnosis Chain API [reference](/api/gnosis/gnosis-api-reference) update.
+* **Documentation**. 
+  * Gnosis Chain API [reference](/api/gnosis/gnosis-api-reference) update.
+  * Debug and trace methods are added to the Ethereum API [reference](/api/ethereum/ethereum-api-reference).
```
Comment (akegaviar) [inline docs/release-notes.md:20]: remove `are` to keep it consistent with the rest of relnotes.

**Review by conor2e — APPROVED** [review]
LGTM

**Comment by akegaviar** [comment]
@ginstagram @soos3d we need:

* A note on how these methods can be accessed with the Chainstack platform. A tip on trace_ being available on Erigon is a start but it's far from all. Think of the user/developer. How can they get all the debug and trace APIs from our platform when they find these API pages? We need a detailed note on both debug_ and trace_ API pages.
* Links on the relevant sections to https://docs.chainstack.com/operations/ethereum/debug-and-trace-apis
* an eth_getBlockReceipts Erigon method + a note how to get the method from our platform.

**Review by conor2e — APPROVED** [review]
LGTM

**Review by akegaviar — APPROVED** [review]

**Review by soos3d — APPROVED** [review]

**Pull request merged**